src/pages/Login/Login.module.css

```diff
@@ -34,3 +34,40 @@
     justify-content: center;
 }
 
+
+.tooltipContainer {
+    position: relative;
+    display: inline-block;
+}
+
+.tooltipContainer .tooltipText {
+    visibility: hidden;
+    width: 120px;
+    background-color: gray;
+    color: #fff;
+    text-align: center;
+    padding: 5px 0;
+    border-radius: 6px;
+    position: absolute;
+    z-index: 1;
+    width: 150px;
+    bottom: 100%;
+    left: 50%;
+    margin-left: -60px;
+}
+
+.tooltipContainer .tooltipText:after {
```

Comment:
I think here and above you don't need to prepend `.tooltipContainer` - there's only one `tooltipText`